src/frontend/src/templates/template.js

```diff
@@ -45,6 +45,10 @@ const useStyles = makeStyles((theme) => ({
     maxWidth: '785px',
     color: theme.palette.text.default,
   },
+  sticky: {
+    position: 'sticky',
+    top: '80px',
+  },
   carousel: {
```

Comment:
1)Find a good position for the carousel, vertically centred. Then after this, we don't want this carousel moving.

2)Let's scroll only the text part.

Reply:
Now the position is moved to the center and fixed. Only text can be scrolled.